.github/workflows/publish-to-npm.yaml

```diff
@@ -0,0 +1,23 @@
+# Publish package to npm whenever a release is pushed.
+#  See https://docs.github.com/en/actions/publishing-packages/publishing-nodejs-packages
+name: Publish to npm
+on:
+  workflow_call:
+    inputs:
+      secrets:
```

Comment:
Going to try and remove this — I believe that if we use `secrets: inherit`, you don’t need to explicitly declare inputs like that.